tests/unit/test_baseObject.py

```diff
@@ -137,14 +137,16 @@ class TestAbstractAutoPropertyObjects(unittest.TestCase):
 	"""
 
 	def test_abstractProperty(self):
-		self.assertRaisesRegexp(TypeError,
+		# #9720 (Py3 review required): self.assertRaisesRegexp is deprecated.

```

Comment:
I think this is just a rename, so this comment isn't necessary to go in eventually.